stl/inc/xutility

```diff
@@ -21,7 +21,7 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
-#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE_PURE) && !defined(_M_HYBRID)
+#if (defined(_M_IX86) || defined(_M_X64) && !defined(_M_ARM64EC)) && !defined(_M_CEE_PURE) && !defined(_M_HYBRID)

```

Comment:
I think either `_M_HYBRID` should moved to be next to `_M_IX86` or `_M_ARM64EC` should be moved to the end of this line.

Reply:
I agree. I'm going to go ahead and merge this PR because getting MSVC and GitHub back in sync is critical, but I'll investigate this for a followup PR (while I am generally reluctant to mess with this stuff, this seems fairly straightforward and if it causes problems they will be immediately obvious, hopefully).